src/Data/Bytes/Builder/Unsafe.hs

```diff
@@ -204,7 +205,13 @@ copyReverseCommits# ::
 copyReverseCommits# _ off Initial s0 = (# s0, off #)
 copyReverseCommits# marr prevOff (Mutable arr sz cs) s0 =
   let !off = prevOff -# sz in
-  case Op.copyMutableByteArray# arr 0# marr off sz s0 of
+  case
```

Comment:
Instead of using CPP here, just reexport `copyMutableByteArray#` in `src-unchecked/Op.hs`, and then you can call `Op.copyMutableByteArray#` here instead without any CPP. The fact that this wasn't already exported from that module was just an oversight.